bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -160,13 +160,13 @@ <h2 class="go">{{ _('Download Firefox') }}</h2>
             <a class="fxos-link" href="{{ url('firefox.desktop.index') }}">{{ _('Learn more') }}</a>
           </div>
         </li>
-      {% if l10n_has_tag('gear_promo_eoy') %}
-        <li id="promo-6" class="item promo-large-landscape gear" tabindex="0">
-          <h2 class="primary go">{{ _('Get the gear, <br>support the mission') }}</h2>
-          <a class="panel-link" rel="external" href="https://gear.mozilla.org?ref=holiday2015&amp;utm_campaign=holiday2015&amp;utm_source=mozilla.org&amp;utm_medium=referral" data-promotion-id="promo-6" data-promotion-name="Get the gear, support the mission" data-promotion-class="promo-large-landscape" data-promotion-type="tile">
+      {% if LANG.startswith('en') %}
+        <li id="promo-6" class="item promo-large-landscape encryption" tabindex="0">
+          <h2 class="primary go">{{ _('Encryption helps <br>protect your privacy') }}</h2>
```

Comment:
Hm yeah, it's not horrible, but it does look better without the break. Will update.